security/zero_trust_workload_identity_manager/zero-trust-manager-overview.adoc

```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="zero-trust-manager-overview"]
-= zero trust workload identity manager for Red{nbsp}Hat OpenShift overview
+= Zero trust workload identity manager for Red{nbsp}Hat OpenShift overview
```

Comment:
It would be great if you can add some specific details about ZTWIM here. We do talk about rest of the SPIFFE components but if you can list some capabilities here it would help users know what this operator is all about. Also, I am guessing that we can still call it out as an operator outside the name.